ja/guide/plugins.md

```diff
@@ -149,6 +222,8 @@ import VueNotifications from 'vue-notifications'
 Vue.use(VueNotifications)
 ```
 
-逆に、サーバーサイドでのみライブラリを読み込む必要がある場合は、`process.server` 変数を使うことができます。これは Webpack が `server.bundle.js` ファイルを作成するタイミングで `true` がセットされる変数です。
+サーバーサイドでのみライブラリを読み込む必要がある場合は、`process.server`変数に `true` がセットされているかでチェックできます。
```

Comment:
`サーバーサイド`をイタリックにしてください

```
原文
 In case you need to import some libraries in a plugin only on *server-side*, you can check if the `process.server` variable is set to `true`.
```